service/src/main/java/com/ericsson/eiffel/remrem/generate/controller/RemremGenerateController.java

```diff
@@ -97,10 +97,12 @@ public void setRestTemplate(RestTemplate restTemplate) {
             @ApiParam(value = "message type", required = true) @RequestParam("msgType") final String msgType,
             @ApiParam(value = "ER lookup result multiple found, Generate will fail") @RequestParam(value = "failIfMultipleFound", required = false, defaultValue = "false") final Boolean failIfMultipleFound,
             @ApiParam(value = "ER lookup result none found, Generate will fail") @RequestParam(value = "failIfNoneFound", required = false, defaultValue = "false") final Boolean failIfNoneFound,
+            @ApiParam(value = RemremGenerateServiceConstants.SHALLOW) @RequestParam(value = "shallow", required = false, defaultValue = "true") final Boolean shallow,
+            @ApiParam(value = RemremGenerateServiceConstants.PAGESIZE) @RequestParam(value = "pageSize", required = false, defaultValue = "1") final int pageSize,
             @ApiParam(value = "JSON message", required = true) @RequestBody JsonObject bodyJson) {
 
         try {
-            bodyJson = erLookup(bodyJson, failIfMultipleFound, failIfNoneFound);
+            bodyJson = erLookup(bodyJson, failIfMultipleFound, failIfNoneFound, shallow, pageSize);
```

Comment:
As I've understood the discussions behind this change the parameters should be called 'connectToExternalERs' and 'limit' instead of 'shallow' and 'pageSize'.

Reply:
Changed accordingly as shallow to connectToExternalERs and pageSize to limit